index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="src/style.css">
+    <title>About RealCoolKids</title>
+</head>
+<body>
+    <h1>About RealCoolKids</h1>
```

Comment:
This is a simple page so not entirely necessary but it's a good idea to get into the habit of putting your header text/images/sometimes nav? in a <header></header> tag for semantics.